syntax = "proto3";

package internal;

message JSONTYPE {
    string json_str = 1;
}